scrapper.py

```diff
@@ -35,7 +48,33 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)AppleWebKit/'
+                          '537.36 (KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36',
+            'accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/'
+                      'avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9',
+            'accept-encoding': 'gzip, deflate, br',
+            'accept-language': 'ru,ru-RU;q=0.9,en-US;q=0.8,en;q=0.7,la;q=0.6'}
+
+        for url in self.seed_urls:
+            response = requests.get(url, headers=headers)
+
+            if not response.ok:
+                print("Request failed")
```

Comment:
it is a weak check. probably we do not want to process such an URL then? what do you think?